dataset/download.htm

```diff
@@ -84,3 +84,48 @@ <h1>3. MASK API</h1>
   <div class="jsonkfunc">frBbox</div><div class="jsonvtxt">Convert bounding boxes to encoded masks.</div>
   <div class="jsonkfunc">frPoly</div><div class="jsonvtxt">Convert polygon to encoded mask.</div>
 </div>
+
+<h1>4. FiftyOne</h1>
+<p>We have partnered with the team behind <a href="https://fiftyone.ai">FiftyOne</a>, an
```

Comment:
FiftyOne provides open-source tool to visualize and facilitate access to COCO data resources and an evaluation tool for model analysis on COCO.